README.md

```diff
@@ -94,4 +105,85 @@ Test(<TestComponent/>)
 
 ~~~
 
+## Find
+
+The find middleware allows you to retrieve a single or multiple elements from the DOM.
+The current find context is kept between calls so you can chain them and find nested elements.
+
+Let's say that you have such a component, rendering two tables:
+
+~~~html
+<div className="component">
+ <table className="table1">
+   <tr><td>1</td><td>2</td></tr>
+   <tr><td>3</td><td>4</td></tr>
+ </table>
+ <table className="table2">
+   <tr><td>5</td><td>6</td></tr>
+   <tr><td>7</td><td>8</td></tr>
+ </table>
+</div>
+~~~
+
+By default, you will always get the first elements matching your search:
+
+~~~js
+Test(<TableComponent/>)
+  .find('table')
+  .find('td')
+  .element(td => {
+    expect(td.getDOMNode().textContent).to.equal('1')
+  })
+~~~
+
+But if you use the `multi` option, you can find nested elements:
```

Comment:
You should make multi the default, like you said. Also, I don't think I like the root option. The component instance is always available inside the test method if you need it